packages/api/src/external/carequality/organization.ts

```diff
@@ -0,0 +1,45 @@
+import { getEnvVarOrFail } from "@metriport/core/util/env-var";
+import { makeCarequalityAPI } from "./api";
+import { buildOrganizationFromTemplate } from "./organization-template";
+import BadRequestError from "@metriport/core/util/error/bad-request";
+
+const cqOrgDetails = {
+  orgName: getEnvVarOrFail("CQ_ORG_NAME"),
+  orgOID: getEnvVarOrFail("CQ_ORG_OID"),
+  addressLine1: getEnvVarOrFail("CQ_ADDRESS_LINE_1"),
+  city: getEnvVarOrFail("CQ_CITY"),
+  state: getEnvVarOrFail("CQ_STATE"),
+  postalCode: getEnvVarOrFail("CQ_POSTAL_CODE"),
+  latitude: getEnvVarOrFail("CQ_LATITUDE"),
+  longitude: getEnvVarOrFail("CQ_LONGITUDE"),
+  urlXCPD: getEnvVarOrFail("CQ_URL_XCPD"),
+  urlDQ: getEnvVarOrFail("CQ_URL_DQ"),
+  urlDR: getEnvVarOrFail("CQ_URL_DR"),
+  contactName: getEnvVarOrFail("CQ_TECHNICAL_CONTACT_NAME"),
+  phone: getEnvVarOrFail("CQ_TECHNICAL_CONTACT_PHONE"),
+  email: getEnvVarOrFail("CQ_TECHNICAL_CONTACT_EMAIL"),
+};
```

Comment:
Now that I look at this, do we need individual env vars for each? Couldn't we just define those on the config files as objects/json, since the config files are TS?

Then it would be as simple as `JSON.parse(getEnvVarOrFail("CQ_ORG_DETAILS"))`